client/reader/site-subscriptions-manager/reader-site-subscriptions.tsx

```diff
@@ -28,15 +31,13 @@ const setUrlQuery = ( key: string, value: string ) => {
 
 	// Only trigger a page show when path has changed.
 	if ( nextPath !== path ) {
-		page.show( nextPath );
+		page.replace( nextPath );
```

Comment:
Here we want to just update the URL to reflect the current search term value, without causing any navigation. We are already on the page where we want to be.